hooks/templatedatavalidator.js

```diff
@@ -63,15 +67,14 @@ function isPageVerticalConfigValid(pageData, jamboConfig) {
  * @param {string} verticalName
  * @param {string} themeDir
  * @param {string} template
+ * @param {Object<string, Function|string>} partials mapping of partial name to partial
  * @returns {boolean}
  */
-function isUniversalSectionTemplateValid(verticalName, themeDir, template) {
-  if (template) {
+function isUniversalSectionTemplateValid(verticalName, themeDir, template, partials) {
+  if (template && !partials[`universalsectiontemplates/${template}`]) {
     const universalSectionPath = path.join(themeDir, 'universalsectiontemplates/', template + '.hbs');
```

Comment:
I felt like it still might be nice for Hitchhikers, would changing it to something like `Could not find universalsectiontemplates/blah. By default this partial is located at ${path1} and ${path2}` work? I'm also fine just changing it to `Could not find partial ~blah~`, but I thought Yen's original error message could be nice for the cases when a Hitchhiker is just trying to use a built in template, rather than more custom cases.

Reply:
Yeah, I like the first thing you suggested. We let them know the key problem: there is no partial with that name, then provide the extra info. Consulting can ignore the extra info if they want.

Reply:
updated! I also added a prefix of the config file, instead of specifying it manually in the msg

Reply:
Changed my mind, don't really want to risk some tiny chance of clobbering somebody's data